extensions/wikia/FounderEmails/FounderEmails.php

```diff
@@ -53,33 +48,36 @@
  */
 $wgExtensionFunctions[] = 'wfFounderEmailsInit';
 
-function wfFounderEmailsInit() {
-	global $wgHooks, $wgAutoloadClasses, $wgFounderEmailsEvents, $wgDefaultUserOptions, $wgCityId;
+function wfFounderEmailsInit()
+{
```

Comment:
Do we actually still need this setup function?

Reply:
It's `kinda` MW extensions.
If (somehow) it's managed somewhere else (order of execution/logs/disable init) then I will leave it as is